config/sidekiq.yml

```diff
@@ -0,0 +1,3 @@
+---
+:queues:
+  - default
```

Comment:
I notice in a few of our other repos, we set a `:logfile: ./log/sidekiq.json.log`. Is that something we want here? I'm guessing the implications of not having it means we don't explicitly log to file (based on https://github.com/alphagov/govuk_sidekiq/blob/master/lib/govuk_sidekiq/sidekiq_initializer.rb#L32), but I don't know our sidekiq setup very well tbh.

Reply:
I'm not sure what sidekiq logs by default, given that this new worker isn't making any logging calls, but I guess adding a `:logfile:` can't hurt